sklearn/preprocessing/imputation.py

```diff
@@ -374,3 +385,352 @@ def transform(self, X):
             X[coordinates] = values
 
         return X
+
+
+class MICEImputer(BaseEstimator, TransformerMixin):
+    """Basic implementation of MICE package from R.
```

Comment:
I would not mention this in the short description. I would say MICE imputation transformer to replace missing value.
It can be the good place to remove the MICE acronym.